Vagrantfile

```diff
@@ -266,8 +266,12 @@ Vagrant.configure('2') do |config|
   config.vm.define :s_standalone do |cfg|
     cfg.vm.provision :chef_client do |chef|
       chef_defaults chef, :s_standalone, 'splunk_standalone'
+# This is used to test installing the forwarder ontop of a node that already has splunk installed. Only have 1 chef_defaults line active.
+#      chef_defaults chef, :s_standalone, 'splunkf_standalone'
       chef.add_recipe 'cerner_splunk_test::install_libarchive'
       chef.add_recipe 'cerner_splunk::server'
+# This is used to test installing the forwarder ontop of a node that already has splunk installed. Only have splunk_server OR forwarder active.
```

Comment:
For consistency, we should probably add a blurb to the docs/vagrant.md file around how to test installing a forwarder along side the full server.

Reply:
Added: https://github.com/cerner/cerner_splunk/pull/265/commits/75111f9aeb55cce020b4a7a2d71e93adc82fc1db